common/src/main/java/org/opensearch/ml/common/transport/upload_chunk/MLUploadModelChunkAction.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.common.transport.upload_chunk;
+
+
+import org.opensearch.action.ActionType;
+
+public class MLUploadModelChunkAction extends ActionType<MLUploadModelChunkResponse> {
+    public static MLUploadModelChunkAction INSTANCE = new MLUploadModelChunkAction();
+    public static final String NAME = "cluster:admin/opensearch/ml/upload_custom_model_chunk";
```

Comment:
We are going to support build-in models in future. How about changing to a more generic name `cluster:admin/opensearch/ml/upload_model_chunk`